### PR TITLE
Update procedure for configuring IPsec

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1666988

### CODE REVIEW

**Comment by jboxman** [comment]
This will get overridden with a PR that explains how to do this for 3.11.

**Comment by jboxman** [comment]
@vikram-redhat, plan is to merge this this week, if SME can verify the updated procedure is correct.

File: admin_guide/ipsec.adoc [hunk 1]
```diff
@@ -39,28 +39,40 @@ with a host that does.
...
- xref:../install_config/configuring_sdn.adoc#configuring-the-pod-network-on-nodes[Configure
- the pod network on nodes] section for information on how to configure the MTU
- to allow space for the IPsec header. This topic describes an IPsec
- configuration that requires 62 bytes. If the cluster is operating on an
- Ethernet network with an MTU of 1500 then the SDN MTU must be 1388, to allow
- for the overhead of IPsec and the SDN encapsulation.
+* Ensure that *libreswan* 3.15 or later is installed on cluster hosts.
+If opportunistic group functionality is required, then *libreswan* version 3.19 or later is required.
+
+* This topic describes an IPsec configuration that requires `62` bytes.
```
Comment (bmcelvee) [inline admin_guide/ipsec.adoc:45]: s/bytes, to/bytes to

**Comment by bmcelvee** [comment]
Just one tiny nit, otherwise LGTM

**Comment by jboxman** [comment]
@bmcelvee thanks!

**Pull request merged**